examples/solvers/forward_backward_primal_dual_denoising.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Total variation denoising using the Forward-backward solver.
```

Comment:
This example uses the partition that `f` are the constraints, `g` is the TV term, and `h` contains the smooth l2-squared term. With deconvolution you need to have the l2-squared term as one of the `g_i` terms, since you need a linear operator acting on `x`.

I can add a deconvolution example as well. More or less the same example as for the Douglas-Rachford?